sample-app/screens/HomeScreen.js

```diff
@@ -1,41 +1,97 @@
 // @flow
 import React from "react";
-import { Button, Platform, StyleSheet, Text, View } from "react-native";
+import {
+  Button,
+  Platform,
+  StyleSheet,
+  Text,
+  View,
+  TouchableHighlight,
+  Image,
+  requireNativeComponent
+} from "react-native";
 import { showMessage } from "../utils";
 import {
   RNJudo,
   type JudoOptions,
   type JudoApplePayOptions,
-  type JudoGooglePayOptions
+  type JudoGooglePayOptions,
+  type PaymentOptions
 } from "judo-react-native";
 
 type Props = {};
 type State = {
   canUseApplePay: boolean,
   canUseGooglePay: boolean
 };
-
+const NativePayButton = requireNativeComponent(
+      'RNNativePayButton',
+    );
 const judoOptions: JudoOptions = {
-  token: "<API_TOKEN>",
-  secret: "<API_SECRET>",
+  token: "<TOKEN>",
+  secret: "<SECRET>",
   judoId: "<JUDO_ID>",
   isSandbox: true,
   amount: "0.01",
   currency: "GBP",
-  consumerReference: "myCustomerReference"
+  consumerReference: "myCustomerReference",
+  paymentReference: "myPaymentReference",
+  metaData: {metadatakey: "metadataValue", metadatakey2: "metadataValue2"}
+};
+
+const ApplePayTransactionType = {
```

Comment:
These enum type things should be exposed as constants from the native side (or imported from judo-react-native) instead of requiring the client app to implement (copy/paste) them.

Reply:
👍